packages/dashboard/src/pages/Wallets/components/AddTokenDialog/index.tsx

```diff
@@ -0,0 +1,309 @@
+import { memo, useEffect, useMemo, useState } from 'react'
+import { MaskColorVar, MaskDialog } from '@dimensiondev/maskbook-theme'
+import { Box, Button, DialogActions, DialogContent, makeStyles, Stack, TextField, Typography } from '@material-ui/core'
+import { EthereumAddress } from 'wallet.ts'
+import { ERC20TokenDetailed, useERC20TokenBalance, useERC20TokenDetailed, useWallet } from '@dimensiondev/web3-shared'
+import { TokenIcon } from '../TokenIcon'
+import { useUpdateEffect } from 'react-use'
+import { useSnackbarCallback } from '../../../../hooks/useSnackbarCallback'
+import { PluginServices } from '../../../../API'
+import { useDashboardI18N } from '../../../../locales'
+
+const useStyles = makeStyles((theme) => ({
+    content: {
+        padding: theme.spacing(3.5, 5, 5),
+        minWidth: 600,
```

Comment:
![image](https://user-images.githubusercontent.com/5390719/121763159-594c2100-cb6c-11eb-8213-4e6dcefc5a12.png)

Please don't.

Reply:
This is bug in storybook